### PR TITLE
fix: custom qml receipts loading, resolve #652

### LINKED ISSUE #652: Nefunkční výběr lístečku ve verzi 2.4.2
Ahoj, zistili jsme, že ve verzi 2.4.2, kterou jsme používali pro generování startovek v 24h formátu, nefunguje výběr jiného typu lístečku (receip) než default. Už standardně používáme vlastní upravené qml a tak jsem musel přeskakovat mezi verzemi. @otahirs
vyberListecku_v2 4 2
vyberListecku_v2 3 9

Díky
Lukáš

### PR DESCRIPTION
**Pull request by otahirs**

### CODE REVIEW

File: quickevent/app/quickevent/plugins/Receipts/src/receiptswidget.cpp [hunk 1]
```diff
@@ -289,18 +289,24 @@ void ReceiptsWidget::markAsPrinted(int connection_id, int card_id)
...
-		QString path = fi.filePath();
-		QString name = path.mid(receipts_dir.length() + 1);
-		if(name.startsWith("private/", Qt::CaseInsensitive))
-			continue;
-		name = name.mid(0, name.indexOf(".qml", Qt::CaseInsensitive));
-		qfInfo() << "receipt:" << name << "path:" << path;
-		ui->lstReceipt->addItem(name, path);
+	QVector<QString> receipts_paths = {
+		getPlugin<ReceiptsPlugin>()->qmlDir() + "/receipts",
+		QCoreApplication::applicationDirPath() + "/receipts",
```
Comment (fvacek) [inline quickevent/app/quickevent/plugins/Receipts/src/receiptswidget.cpp:294]: We have `quickevent-data` directory introduced for custom app style-sheet, custom receipts should be IMO put there as well.
```c++
	QCoreApplication::applicationDirPath() + "/quickevent-data/receipts",
	QDir(".").absolutePath() + "/quickevent-data/receipts"
```

Comment (otahirs) [reply quickevent/app/quickevent/plugins/Receipts/src/receiptswidget.cpp:294]: ok.
offtopic: any idea why is the `sound` folder inside `style` folder?

Comment (fvacek) [reply quickevent/app/quickevent/plugins/Receipts/src/receiptswidget.cpp]: because sounds can be customized as well, it is not visual style only

Comment (otahirs) [reply quickevent/app/quickevent/plugins/Receipts/src/receiptswidget.cpp]: but why this folder structure, sounds are not related to style(sheets),  just `/quickevent-data/sounds` seems more logical to me
resulting in
`/quickevent-data/styles`
`/quickevent-data/sounds`
`/quickevent-data/receipts`

Comment (fvacek) [reply quickevent/app/quickevent/plugins/Receipts/src/receiptswidget.cpp]: well, `/quickevent-data/sound` sounds better, please create MR if you think that it worth to do it.

**Pull request merged**